plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/utils/linux/KVMHostInfo.java

```diff
@@ -248,6 +248,6 @@ private void getHostInfoFromLibvirt() {
 
     private String getCPUArchFromCommand() {
         LOGGER.info("Fetching host CPU arch");
-        return Script.runSimpleBashScript(cpuArchCommand);
+        return Script.runSimpleBashScript(Script.getExecutableAbsolutePath(cpuArchRetrieveExecutable));
```

Comment:
```suggestion
        return Script.runSimpleBashScript(Script.getExecutableAbsolutePath(cpuArchRetrieveExecutable), 1000);
```

increase timeout required here as well?

Reply:
@sureshanaparti as this was already running without a timeout and I didn't face an issue here, I have not changed this